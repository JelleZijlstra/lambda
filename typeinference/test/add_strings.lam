let s1 = "hello"
let s2 = "world"
;;

print (add_strings s1 s2)