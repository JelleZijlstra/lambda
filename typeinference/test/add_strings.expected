%%stdout%%
"helloworld"
Result: module
	let s2 = "world"
	let s1 = "hello"
	let __result__ = ()
end
%%stderr%%
